pkg/util/workqueue/queue.go

```diff
@@ -113,6 +113,17 @@ func (q *Type) Get() (item interface{}, shutdown bool) {
 	return item, false
 }
 
+// Get the head of the queue
+func (q *Type) Peek() interface{} {
+	q.cond.L.Lock()
+	defer q.cond.L.Unlock()
+	if len(q.queue) == 0 {
+		return nil
+	}
+	item := q.queue[0]
+	return item
```

Comment:
No need to create a new var just to return it immediately afterwards, just `return q.queue[0]`.